pkg/version/version.go

```diff
@@ -56,14 +54,13 @@ type MeshInfo []ServerInfo
 // NodeType decides the responsibility of the proxy serves in the mesh
 type NodeType string
 
-func ToUserFacingNodeType(t model.NodeType) NodeType {
+func ToUserFacingNodeType(t string) NodeType {
 	switch t {
-	case model.Router:
```

Comment:
maybe add some comments, otherwise someone will try to refactor this in the future.